apps/myaccount/src/app.tsx

```diff
@@ -261,12 +270,13 @@ export const App = (): ReactElement => {
                                                         ? (
                                                             <link
                                                                 href={
+                                                                    // eslint-disable-next-line max-len
                                                                     `${window?.origin}${window?.publicPath}/libs/themes/${theme}/theme.${window?.themeHash}.min.css`
```

Comment:
Do not supress.
```suggestion
                                                                    `${window?.origin}${window?.publicPath}/libs/themes/${
                                                                        theme
                                                                     }/theme.${window?.themeHash}.min.css`
```

Reply:
Fixed from https://github.com/wso2/identity-apps/pull/3447/commits/55be9747622174e33bd5818d10159668b9761625